playbooks/capsule-dev/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
We could turn the parts that are the same into their own task files and the parts that are different, vary those depending on a variable passed in.